doc/examples/index.rst

```diff
@@ -7,3 +7,4 @@ Examples
    introduction
    dask_support
    custom_indexes
+   query
```

Comment:
`pooch` is missing in `environment.yml` to load the xarray tutorial dataset in this example notebook.